contracts/integrations/custom/CustomIntegrationLiquity.sol

```diff
@@ -0,0 +1,232 @@
+     * @return uint256                   Call value
+     * @return bytes                     Trade calldata
+     */
+    function _getExitCalldata(
+        address _strategy,
+        bytes calldata, /* _data */
+        uint256, /* _resultTokensIn */
+        address[] calldata, /* _tokensOut */
+        uint256[] calldata /* _minAmountsOut */
+    )
+        internal
+        view
+        override
+        returns (
+            address,
+            uint256,
+            bytes memory
+        )
+    {
+        return (address(liquityBorrowerOperations), 0, abi.encodeWithSelector(
+          IBorrowerOperations.closeTrove.selector
+        ));
+    }
+
+    /**
+     * The list of addresses of the IERC-20 tokens mined as rewards during the strategy
+     *
+     * hparam  _data                      Address provided as param
+     * @return address[] memory           List of reward token addresses
+     */
+    function _getRewardTokens(
+        address /* _data */
+    ) internal pure override returns (address[] memory) {
+        // no LQTY rewards for lending or borrowing
+        return new address[](0);
+    }
+
+    /* ============ External Functions ============ */
+
+    /**
+     * The tokens to be purchased by the strategy on enter according to the weights.
+     * Weights must add up to 1e18 (100%)
+     *
+     * hparam  _data                      Address provided as param
+     * @return _inputTokens               List of input tokens to buy
+     * @return _inputWeights              List of weights for the tokens to buy
+     */
+    function getInputTokensAndWeights(
+        bytes calldata /* _data */
+    ) external pure override returns (address[] memory _inputTokens, uint256[] memory _inputWeights) {
+        address[] memory inputTokens = new address[](1);
+        // inputTokens[0] is address(0) (ether)
+        uint256[] memory inputWeights = new uint256[](1);
+        inputWeights[0] = 1e18; // 100%
+        return (inputTokens, inputWeights);
+    }
+
+    /**
+     * The tokens to be received on exit.
+     *
+     * hparam  _strategy                  Strategy address
+     * hparam  _data                      Bytes data
+     * hparam  _liquidity                 Number with the amount of result tokens to exit
+     * @return exitTokens                 List of output tokens to receive on exit
+     * @return _minAmountsOut             List of min amounts for the output tokens to receive
+     */
+    function getOutputTokensAndMinAmountOut(
+        address _strategy,
+        bytes calldata, /* _data */
+        uint256 /* _liquidity */
+    ) external view override returns (address[] memory exitTokens, uint256[] memory _minAmountsOut) {
+        address[] memory outputTokens = new address[](1);
+        // outputTokens[0] is address(0) (ether)
+        uint256[] memory outputAmounts = new uint256[](1);
+        outputAmounts[0] = _getCollateral(_strategy); // TODO: deal with fees
```

Comment:
Actually - I think the fees are taken in LUSD, so we don't need to do anything here. Will update....

Reply:
the output amount should be the LUSD amount right and not the value of the collateral

Reply:
The output token is Ether, so this should be the value of our Ether collateral, no?

Reply:
correct, naming is a bit confusing xD

Reply:
This function is simple, just return the current collateral amount.